modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -43,6 +43,7 @@
             <class name="org.wso2.am.integration.tests.oas.OASTestCase"/>
             <class name="org.wso2.am.integration.tests.api.lifecycle.APIVisibilityByPublicTestCase"/>
             <class name="org.wso2.am.integration.tests.api.lifecycle.APIVisibilityByDomainTestCase"/>
+            <class name="org.wso2.am.integration.tests.api.lifecycle.EditAPIAndCheckUpdatedInformationTestCase"/>
```

Comment:
We are planning to include all the tests at once in the testng file. Shall we remove this for now?

Reply:
Fixed